src/vs-bicep/Bicep.VSLanguageServerClient.TestServices/LanguageServerActivationService.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Linq;
+using Bicep.VSLanguageServerClient.TestServices.Utilitites;
+using Microsoft.Test.Apex.VisualStudio;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.Shell;
+
+namespace Bicep.VSLanguageServerClient.TestServices
+{
+    // ILanguageServiceBroker2.ActiveLanguageClients is marked obsolete. We will replace it when an alternate option is available.
```

Comment:
This api is going to be redesgined. It's used in few other projects and we are okay to use till an alternate solution is available. Please ping me for more details.